jwt.go

```diff
@@ -30,7 +40,10 @@ func (h JWTAuth) ServeHTTP(w http.ResponseWriter, r *http.Request) (int, error)
 		if err != nil {
 			return handleUnauthorized(w, r, p), nil
 		}
-		vClaims := vToken.Claims.(jwt.MapClaims)
+		vClaims, err := flatten.Flatten(vToken.Claims.(jwt.MapClaims), "", flatten.DotStyle)
```

Comment:
This may conflict with the PR I merged yesterday that implements a "contains" rule.  The new rule lets you specify `allow` and `deny` rules based on the presence of a string in a JSON array.  For example:

```
...
allow role admin
...
```

will work for a JWT with the following role array:

```
role: [ "admin", "user" ]
```

I think the contains rule will have to be rewritten to accommodate the way arrays are flattened.